src/components/solverComponents/SAT_Components/SAT_TextArea.tsx

```diff
@@ -0,0 +1,42 @@
+import 'prismjs/themes/prism-solarizedlight.css'
+import React from "react";
+import Editor from "react-simple-code-editor";
+import { highlight } from 'prismjs/components/prism-core';
+import { Container } from "@chakra-ui/react";
+import { SAT_language } from "./prism-SAT.js"
+
+interface ISAT_TextAreaProps {
+
+}
+
+interface ISAT_TextAreaState {
+    problemString: string;
+}
+
+export class SAT_TextArea extends React.Component<ISAT_TextAreaProps, ISAT_TextAreaState> {
+    constructor(props: ISAT_TextAreaProps) {
+        super(props);
+        this.state = { problemString: "" };
+    }
+
+    setProblemString(code: string): void {
+        this.setState({problemString: code});
+    }
+
+    render() {
+        return (
+        <Container border="1px" borderColor="#AAAAAA" borderRadius="10px">
```

Comment:
The editor shows up in a black box inside the container, can you remove the black rectangle? (see below)

Also, please use the [theme system of chakra ui](https://chakra-ui.com/docs/styled-system/theme) for [colors, border properties, etc.](https://chakra-ui.com/docs/styled-system/style-props).

![image](https://user-images.githubusercontent.com/60352933/192228606-1cd3a6c1-d273-4964-9f5a-e1390d92ed5e.png)

Reply:
to use a custom color palette with primsjs is complicated, maybe visit later